modules/ROOT/pages/permissions-by-product.adoc

```diff
@@ -34,111 +51,296 @@ Or:
+
+DataGraph Project - Admin::  Enables a user to: 
++
+* Contribute, consume, and operate the unified schema.
+* View usage metrics in a specific project.
+
+== Design Center
 
-* Project Administrator
-* Project Editor
-* Project Viewer
+Design Center Developer:: Enables a user to view, create, and manage all projects within a business group. 
++
+Use this permission to set up administrators for all projects within a specific business group.
+Design Center Creator:: Enables a user to create projects in Design Center from the navigation panel and view all projects created or shared with the user. 
++
+Use this permission to invite users to create, edit, and maintain your projects.
+Design Center Viewer:: Enables a user to view all Design Center projects within a business group and test projects with the Mocking Service. +
+Users with this permission cannot create new projects, edit or rename existing projects, or share projects with another user. 
++
+Assign this permission to those who consume your project in a specific business group.
+
+Design Center also has project level permissions: 
+
+Project Administrator:: Enables a user to manage and share a Design Center project within a business group. 
++
+Use this permission to set up administrators for all the projects within a specific business group.
+Project Editor:: Enables a user to edit a Design Center project within a business group. 
++
+Use this permission to invite users to create, edit, and maintain your projects.
+Project Viewer:: Enables a user to view a Design Center project within a business group and test projects with the Mocking Service. 
++
+Users with this permission cannot create a new project, edit or rename the existing project, or share the project with another user. 
++
+Assign this permission to those who consume your project in a specific business group.
+
+// where did the project-level permissions go?
 
 == Exchange
 
-Organization level permissions:
+Exchange Administrator:: Enables a user to: 
++
+* View, create, and download assets within a business group.
+* Edit asset portal content in an existing asset version.
++
+Users with this permission have all the same access as users with the Exchange contributor and Exchange viewer permissions, and access to share an asset with another user, deprecate an asset, and delete an asset. 
++
+Use this permission to set up Exchange administrators for all assets within a specific business group.
+
+Exchange Contributor:: Enables a user to view, create, and download assets within a business group. 
++
+Users with this permission can edit asset portal content in an existing asset version. 
++
+Use this permission to invite users to edit and maintain your asset portal descriptions.
+
+Exchange Viewer:: Enables a user to view and download assets within a business group. 
++
+Users with this permission cannot add new assets, edit asset portal content, or share an asset with another user. 
```

Comment:
This might be better as the second sentence in the first paragraph. I know I asked you to separate text, but in this case, I think it is tightly-coupled with the initial text. Up to you.